backend/src/main/kotlin/blue/mild/covid/vaxx/setup/ConfigurationDependencyInjection.kt

```diff
@@ -46,4 +48,6 @@ fun DI.MainBuilder.bindConfiguration() {
     bind<String>("frontend") with singleton {
         getEnvOrLogDefault("FRONTEND_PATH", "../frontend/dist/frontend")
     }
+
+    bind<PatientService>("PatientService") with singleton { PatientService(instance()) }
```

Comment:
tenhle file spis pouzivam na konfiguracni veci - vselijake stringy a tak vedle je `DependencyInjection.kt` kam hazim classy

+ je treba jeste tenhle bind udelat pro tu `ValidationService` afaik inicializovana neni -- coz bys zjistil, kdybys to pustil 😉